osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -339,6 +351,45 @@ public virtual IBeatmap GetPlayableBeatmap(IRulesetInfo ruleset, IReadOnlyList<M
 
         #endregion
 
+        #region Audio Normalization
```

Comment:
This will need to be moved to a more global area, likely `MusicController`

Reply:
Would moving the functions into `AudioNormalization` and calling them from `MusicController` suffice?